web/src/features/app-sidebar/AppSidebar.tsx

```diff
@@ -31,34 +32,35 @@ export const SIDEBAR_WIDTH = '63px';
 
 const PORTAL_URL = 'https://portal.electricitymaps.com';
 
-const MENU_ITEMS = [
-  {
-    to: '/',
-    label: 'Map',
-    icon: MapIcon,
-    isActive: true,
-  },
-  {
-    label: 'Datasets',
-    to: `${PORTAL_URL}/datasets`,
-    icon: FileDownIcon,
-  },
-  {
-    label: 'API',
-    to: `${PORTAL_URL}`,
-    icon: CodeXmlIcon,
-  },
-  {
-    label: 'API Docs',
-    to: `${PORTAL_URL}/docs`,
-    icon: BookOpenIcon,
-  },
-];
-
 export function AppSidebar() {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
   const tooltipReference = useRef<TooltipWrapperReference>(null);
   const isIntercomEnabled = useFeatureFlag('intercom-messenger');
+  const { t } = useTranslation();
+
+  const MENU_ITEMS = [
+    {
+      to: '/',
+      label: t('sidebar-menu.map'),
+      icon: MapIcon,
+      isActive: true,
+    },
+    {
+      label: t('sidebar-menu.datasets'),
+      to: `${PORTAL_URL}/datasets`,
+      icon: FileDownIcon,
+    },
+    {
+      label: t('sidebar-menu.api'),
+      to: `${PORTAL_URL}`,
+      icon: CodeXmlIcon,
+    },
+    {
+      label: t('sidebar-menu.api-docs'),
+      to: `${PORTAL_URL}/docs`,
+      icon: BookOpenIcon,
+    },
+  ];
```

Comment:
Please don't move these inside the component or translate them. They are designed to match our data portal which don't use translations as of now.